doc/api/http.md

```diff
@@ -841,8 +841,25 @@ Note that the socket timeout logic is set up on connection, so
 changing this value only affects *new* connections to the server, not
 any existing connections.
 
-Set to 0 to disable any kind of automatic timeout behavior on incoming
-connections.
+Set to 0 to disable timeout behavior on incoming connections.
+If `server.timeout` is not 0 and [`server.keepAliveTimeout`][] is 0,
```

Comment:
IMHO you can simplify to just ```If [`server.keepAliveTimeout`][] is 0, the value of `server.timeout` will be used instead.```
And now it's a *Note:* for `server.keepAliveTimeout`